resource-managers/kubernetes/integration-tests/pom.xml

```diff
@@ -76,6 +76,12 @@
       <artifactId>spark-tags_${scala.binary.version}</artifactId>
       <type>test-jar</type>
     </dependency>
+    <dependency>
+      <groupId>com.amazonaws</groupId>
+      <artifactId>aws-java-sdk</artifactId>
+      <version>1.7.4</version>
```

Comment:
Is this harmonized with the other AWS dependency? may be hard to do so in the build.

Reply:
The one I see is about kinesis  do you mean that? That is an external dependency. We can run build/tests without that profile as they are not related.